amenity-points.mss

```diff
@@ -2064,11 +2064,12 @@
     text-wrap-width: @standard-wrap-width;
   }
 
-  [feature = 'natural_tree'][zoom >= 17] {
+    [feature = 'natural_tree'] {
     text-name: "[name]";
     text-size: @standard-text-size;
     text-fill: green;
-    text-dy: 7;
+    [zoom >= 16][denotation = 'natural_monument'] { text-dy: 6; }
```

Comment:
This is what causes your error - you use the column `denotation` here which means all layers this rule is used on need to provide it - which they don't, since your project.mml changes only modify the tree layers and not the text layers.